ui.tests/test-module/specs/wizard/wizard.locale.spec.js

```diff
@@ -0,0 +1,78 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+const afConstants = require("../../libs/commons/formsConstants");
+const sitesSelectors = require("../../libs/commons/sitesSelectors");
+describe('Locale - Authoring Test', function () {
+    const changeLanguage = function (str) {
+        cy.visit('http://localhost:4502/aem/forms.html/content/dam/formsanddocuments');
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userproperties.html"]').click();
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userpreferences.html"]').click();
+        cy.get('[name="language"].coral-Form-field._coral-Dropdown').first().click();
+        cy.get(`coral-selectlist-item[value=${str}]`).click({force: true});
+        cy.get('._coral-Button._coral-Button--cta[trackingelement="accept"]').click();
+    }
+    context('Test Wizard Component String Language', function () {
+        beforeEach(() => {
+            cy.intercept('http://localhost:4502/editor.html/content/forms/af/language-test.html').as('createFormRequest');
+            cy.intercept("/adobe/forms/fm/v1/templates*").as("templates");
+            cy.visit('http://localhost:4502/aem/start.html');
+            cy.get('#username').type('admin');
+            cy.get('#password').type('admin');
+            cy.get('#submit-button').click();
+            cy.on('uncaught:exception', () => {
+                return false
+            });
+
+            changeLanguage("de");
+            cy.visit('http://localhost:4502/libs/fd/fm/gui/content/forms/af/create.html').then(() => {
+                cy.wait('@templates', {requestTimeout: 30000});
+                cy.get('[data-item-id="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"]').click();
+                cy.contains('button', 'Erstellen').click();
+                cy.get('input[name="submitDialogTitle"]').type('language-test');
+                cy.get('[data-testid="modal"]').contains('button', 'Erstellen').last().click();
+            });
+            cy.visit('http://localhost:4502/editor.html/content/forms/af/language-test.html');
+            const dataPath = "/content/forms/af/language-test/jcr:content/guideContainer/*",
+                responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+            cy.selectLayer("Edit");
+            cy.insertComponent(responsiveGridDropZoneSelector, "Adaptives Formular – Assistenten-Layout", afConstants.components.forms.resourceType.wizard);
+            cy.get('body').click(0, 0);
+        });
+
+        it('Wizard String language test for English', function () {
+            changeLanguage("en");
+            cy.visit("http://localhost:4502/editor.html/content/forms/af/language-test.html").then(() => {
```

Comment:
Please don't hard code URL's make use of openPage commands which already exist. Developers can customize the aem end point or add a context path to the URL, all these details are already abstracted out in openPage